metrics/grafana/tiflow.json

```diff
@@ -9,13 +9,231 @@
       "pluginName": "Prometheus"
     }
   ],
-  "__elements": {},
+  "__elements": {
+    "WBa0-4KVz": {
+      "name": "Uptime",
+      "uid": "WBa0-4KVz",
+      "kind": 1,
+      "model": {
+        "aliasColors": {},
+        "bars": false,
+        "dashLength": 10,
+        "dashes": false,
+        "datasource": {
+          "type": "prometheus",
+          "uid": "${DS_PROMETHEUS}"
+        },
+        "description": "Uptime of TiCDC and TiKV",
+        "fieldConfig": {
+          "defaults": {
+            "links": []
+          },
+          "overrides": []
+        },
+        "fill": 1,
+        "fillGradient": 0,
+        "hiddenSeries": false,
+        "legend": {
+          "alignAsTable": true,
+          "avg": false,
+          "current": true,
+          "max": false,
+          "min": false,
+          "rightSide": true,
+          "show": true,
+          "total": false,
+          "values": true
+        },
+        "lines": true,
+        "linewidth": 1,
+        "links": [],
+        "nullPointMode": "null",
+        "options": {
+          "alertThreshold": true
+        },
+        "paceLength": 10,
+        "percentage": false,
+        "pluginVersion": "9.2.1",
+        "pointradius": 2,
+        "points": false,
+        "renderer": "flot",
+        "seriesOverrides": [],
+        "spaceLength": 10,
+        "stack": false,
+        "steppedLine": false,
+        "targets": [
+          {
+            "datasource": {
+              "type": "prometheus",
+              "uid": "${DS_PROMETHEUS}"
+            },
+            "editorMode": "code",
+            "expr": "(time() - process_start_time_seconds{k8s_cluster=\"$k8s_cluster\", instance=~\"$instance\"})",
+            "format": "time_series",
+            "intervalFactor": 1,
+            "legendFormat": "Tiflow-{{instance}}",
```

Comment:
Is it possible to distinguish whether the role of instance is tiflow master or tiflow executor.

Reply:
Currently, components can only be distinguished by specifying `job_name` at promethues config file.